apps/web-portal-client/lib/features/auth/containers/PrivateRoute.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import {Route, Redirect} from 'react-router-dom'
+import {useSelector} from 'react-redux'
+import {selectAuthToken} from '../selectors'
+
+export function PrivateRoute({children, ...rest}) {
+  const token = useSelector(selectAuthToken)
+  return (
+    <Route
+      // eslint-disable-next-line
```

Comment:
replace with a specific eslint rule disable here

It's ok to turn off eslint rule here, but better to be specific in that case